src/db/sysdb_ops.c

```diff
@@ -3662,12 +3669,13 @@ errno_t sysdb_remove_attrs(struct sss_domain_info *domain,
     return ret;
 }
 
-static errno_t sysdb_search_object_by_str_attr(TALLOC_CTX *mem_ctx,
-                                   struct sss_domain_info *domain,
-                                   const char *filter_tmpl,
-                                   const char *str,
-                                   const char **attrs,
-                                   struct ldb_result **_res)
+static errno_t sysdb_search_object_by_str_attr_ex(TALLOC_CTX *mem_ctx,
+                                                 struct sss_domain_info *domain,
+                                                 const char *filter_tmpl,
+                                                 const char *str,
+                                                 const char **attrs,
+                                                 bool sanitize_input,
```

Comment:
I would prefer if we could backport some sysdb patches to 1.13 instead of introducing boolean parameter.
Boolean parameter does does say something in this context but it does not say a lot when you call function.
And you can easily do copy&paste bug. This might be especially problem when 1.13 will be more diverged from master in future.